biomage/rds/migrate.py

```diff
@@ -0,0 +1,92 @@
+import os
+import subprocess
+from pathlib import Path
+
+import click
+
+from ..utils.AuroraClient import AuroraClient
+from ..utils.constants import DEFAULT_AWS_ACCOUNT_ID, DEVELOPMENT, STAGING
+
+IAC_PATH = os.getenv("BIOMAGE_IAC_PATH")
+
+def _migrate(iac_path, migration_env):
+    proc = subprocess.Popen(
+    [
+        "node_modules/.bin/knex",
+        "migrate:latest",
+        "--cwd",
+        iac_path
+    ],
+        cwd=iac_path,
+        env=migration_env
+    )
+    proc.wait()
+
+@click.command()
+@click.option(
+    "-i",
+    "--input_env",
+    required=True,
+    default=DEVELOPMENT,
+    show_default=True,
+    help="Path to the IAC folder",
+)
+@click.option(
+    "-s",
+    "--sandbox_id",
+    required=False,
+    default=None,
+    show_default=True,
+    help="Sandbox id to migrate to. Required if migrating to staging",
+)
+@click.option(
+    "--iac_path",
+    required=False,
+    default=IAC_PATH,
+    show_default=True,
+    help="Path to the IAC folder",
+)
+def migrate(iac_path, sandbox_id, input_env):
+    """
+    Migrate database in environment
+    """
+
+    if not IAC_PATH:
```

Comment:
I think that most of us have all repositories under the same folder, so maybe the easiest default would be to assume that? I'm just trying to make this work out of the box as opposed to having people needing to set up a new variable (which is not a big deal, but many people will forget or miss it and spend some time figuring it out)

Reply:
if the `iac` repo is not where expected, we could fall back to using `IAC_PATH` you proposed. What do you think?

Reply:
sure, I'll make that change